src/backend/interfaces/REST/py.server/caliopen_api/user/imports.py

```diff
@@ -9,9 +9,6 @@
 from caliopen_main.parsers.vcard import parse_vcards
 from caliopen_main.parsers.vcard import read_file
 
-import logging
-log = logging.getLogger(__name__)
```

Comment:
import is done line 1 and log instanciated line 20 on master. Sound legit to remove this doublon